src/templates/actors/partials/sheet-v2/character-abilities-tab.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="ability-fields ability-fields--exploration-skills {{styles.ability.exploration}}">
+  {{#with actor.system.exploration}}
+    {{> (path "/templates/actors/partials/sheet-v2/components/character-ability-field.hbs")
+      label="OSE.exploration.ld.short"
+      icon=(asset '/Melee.svg')
+      value=ld
+      name="system.exploration.ld"
+      explorationType="ld"
+    }}
+
+    {{> (path "/templates/actors/partials/sheet-v2/components/character-ability-field.hbs")
+      label="OSE.exploration.od.short"
+      icon=(asset '/Melee.svg')
+      value=od
+      name="system.exploration.od"
+      explorationType="od"
+    }}
+
+    {{> (path "/templates/actors/partials/sheet-v2/components/character-ability-field.hbs")
+      label="OSE.exploration.sd.short"
+      icon=(asset '/Melee.svg')
+      value=sd
+      name="system.exploration.sd"
+      explorationType="sd"
+    }}
+
+    {{> (path "/templates/actors/partials/sheet-v2/components/character-ability-field.hbs")
+      label="OSE.exploration.ft.short"
+      icon=(asset '/Melee.svg')
+      value=ft
+      name="system.exploration.ft"
+      explorationType="ft"
+    }}
+  {{/with}}
+</div>
+
+<uft-expandable-section type="ability" can-create aria-expanded>
+  <h2 slot="heading">{{localize "OSE.category.abilities"}}</h2>
```

Comment:
I don't have a lot of imagination for why this has to be expandable. I'll have to go back to the original design to see what that looked like

<img width="587" alt="image" src="https://github.com/vttred/ose/assets/1410433/b5401c05-d77e-4260-9443-9ebcd2fa27a8">